quiz/category/category.html

```diff
@@ -38,19 +38,19 @@ <h2>Start Your Quiz Here</h2>
         <div class="category-container">
             <div class="card category card-box-shadow">
                 <img class="image-resp" src="../../images/quiz.png" alt="image">
-                <a href="#">
+                <a href="../rules/rules.html">
                     <button class="btn btn-primary width-100">Play now</button>
                 </a>
             </div>
             <div class="card category card-box-shadow">
                 <img class="image-resp" src="../../images/quiz.png" alt="image">
```

Comment:
proper alt name can be given, it can be done later , rest of the code looks good
A UI suggestion would be to make the start quiz button a bit big.

Reply:
I will change the alt name later.